digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/HouseholdRegistration.java

```diff
@@ -5,50 +5,17 @@
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-
-import java.util.List;
+import org.egov.common.contract.request.RequestInfo;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 public class HouseholdRegistration extends CampaignData {
```

Comment:
This model looks very similar now to health.sync.web.models.request.HouseholdRegistrationRequest. Do we need the requestInfo here?